lib/models/Student.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'Person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age:, name: 'Unknown', parent_permission: true, classroom: 'Unknown')
+    super(age, name: name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
@OyePriscilla It is not missing. The age you entered is 33, but in the example, it should be 17. If you check it with the correct number, it will prompt for permission. It takes about 15 seconds to check the conditional statement.

Reply:
@romans-adi Ok, let me confirm that. But I'm thinking, was it stated in the requirement if there is age limit for the student class?

Reply:
When I entered digits lesser than 17, it worked fine; but I think there is no limitation as regards the student's age. More so, when I entered a wrong answer; the app still worked, this is against the project objective. 👍 

<img width="428" alt="Untitled" src="https://github.com/PabloEGonz/school-library/assets/103386183/c812c060-e01e-46e8-9fb0-2203fce2427e">

Reply:
@OyePriscilla Agree. Thank you for your time :) We will fix that as soon as possible!

Reply:
@romans-adi You are most welcome. 👍